HelloFlask/templates/hello_there.html

```diff
@@ -2,10 +2,10 @@
 <html>
     <head>
         <meta charset="utf-8" />
-        <link rel="stylesheet" type="text/css" href="/static/site.css" />
+        <link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='site.css') }}" />
         <title>{{ title }}</title>
     </head>
     <body>
-        <span class="message">{{ message }}</span> It's {{ date }}.
+        <span class="message">Hello there, {{ name }}!</span> It's {{ date.strftime("%A, %d %B, %Y at %X") }}.
     </body>
 </html>
```

Comment:
file doesn't have an ending linebreak. not pretty when you `cat` such a file in a terminal 😿

sounds like VS code actually has an option for this: https://github.com/Microsoft/vscode/issues/1666
dunno if it's enabled by default though (I hope it is)

Reply:
"files.insertFinalNewline": true is the setting, but it's false by default. :(